subsys/bluetooth/host/conn.c

```diff
@@ -1471,6 +1471,12 @@ struct net_buf *bt_conn_create_pdu_timeout(struct net_buf_pool *pool,
 	 */
 	__ASSERT_NO_MSG(!k_is_in_isr());
 
+	if (!K_TIMEOUT_EQ(timeout, K_NO_WAIT) &&
+	    k_current_get() == k_work_queue_thread_get(&k_sys_work_q)) {
+		LOG_DBG("Blocking the SYSWQ is bad, mmkay?");
```

Comment:
Mr. Mackey says that we should have proper debug logs
```suggestion
```

IMO this does not require a log statement at all

Reply:
I think it does since it's a "silent" override. I would be confused at least that my supposedly-blocking alloc is non blocking.
I tried to look into dumping the `timeout` value in the warning below but it's opaque and has no getters in zephyr :/
I'll reword it ofc

Reply:
That is a fair point. I think we should add a similar LOG statement in `bt_att_req_alloc` as well then

Reply:
will do :+1:

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/72271